frontend/src/components/Dashboard/SideBar.js

```diff
@@ -0,0 +1,261 @@
+                                            }
+                                        >
+                                            {state}
+                                        </a>
+                                        {': ' + bundlesDict[state]}
+                                    </li>,
+                                );
+                            }
+                        }
+                        this.setState({ bundles: bundles });
+                    }
+                },
+                error: (xhr, status, err) => {
+                    console.error(xhr.responseText);
+                },
+            });
+        };
+        // Start to fetch the bundles' count
+        fetchBundles(0, {});
+    }
+
+    render() {
+        const { classes, userInfo } = this.props;
+        if (!userInfo) {
+            return null;
+        }
+        return (
+            <Card className={cx(classes.card)} elevation={0} style={{ height: '100%' }}>
+                <Avatar src={'https://i.pravatar.cc/30'} className={classes.avatar} />
+                <Box className={classes.box}>
+                    <h3 className={classes.name}>{userInfo.user_name}</h3>
+                    {userInfo.affiliation ? (
+                        <p className={classes.affiliation}>Affiliation: {userInfo.affiliation}</p>
+                    ) : null}
+                </Box>
+                <Divider />
+
+                {this.props.showQuota ? (
+                    <Box className={classes.box}>
+                        <a
+                            className={classes.subheader}
+                            href={
+                                'https://github.com/codalab/codalab-worksheets/blob/master/docs/FAQ.md#how-do-i-reduce-the-amount-of-disk-usage'
+                            }
+                        >
+                            Disk Usage
+                        </a>
+                        <Box className={classes.progressBox}>
+                            <BorderLinearProgress
+                                className={classes.progress}
+                                variant='determinate'
+                                color='secondary'
+                                value={(userInfo.disk_used / userInfo.disk_quota) * 100}
+                            />
+                            <span className={classes.value}>{renderSize(userInfo.disk_quota)}</span>
+                        </Box>
+                        <span className={classes.value}>
+                            {Math.floor((userInfo.disk_used / userInfo.disk_quota) * 100) + '%'}
+                        </span>
+                        <br />
+                        <a
+                            className={classes.subheader}
+                            href={
+                                'https://github.com/codalab/codalab-worksheets/blob/master/docs/FAQ.md'
```

Comment:
Same here with FAQ (link to the page on https://codalab-worksheets.readthedocs.io/)